x-pack/test/functional/apps/transform/creation_saved_search.ts

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+interface GroupByEntry {
+  identifier: string;
+  label: string;
+  intervalLabel?: string;
+}
+
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const transform = getService('transform');
+
+  describe('creation_saved_search', function() {
+    this.tags(['smoke']);
+    before(async () => {
+      await esArchiver.load('ml/farequote');
+    });
+
+    after(async () => {
+      await esArchiver.unload('ml/farequote');
+      await transform.api.cleanTransformIndices();
+    });
+
+    const testDataList = [
+      {
+        suiteTitle: 'batch transform with terms groups and avg agg with saved search filter',
+        source: 'farequote_filter',
+        sourceIndex: 'farequote',
+        sourcePreviewColumn: 3,
+        sourcePreviewExpectedColumnValues: 'ASA',
+        pivotPreviewColumn: 0,
+        pivotPreviewExpectedColumnValues: 'ASA',
+        groupByEntries: [
+          {
+            identifier: 'terms(airline)',
+            label: 'airline',
+          } as GroupByEntry,
+        ],
+        aggregationEntries: [
+          {
+            identifier: 'avg(responsetime)',
+            label: 'responsetime.avg',
+          },
+        ],
+        transformId: `fq_1_${Date.now()}`,
+        transformDescription:
+          'farequote batch transform with groups terms(airline) and aggregation avg(products.base_price) with saved search filter',
```

Comment:
Typo - should refer to the avg(responsetime) agg I think.

Reply:
Fixed in a2fc19a.